src/universal/modules/userDashboard/containers/UserSettings/UserSettingsContainer.js

```diff
@@ -19,21 +21,24 @@ const updateSuccess = {
 };
 
 const mapStateToProps = (state) => {
+  const userId = state.auth.obj.sub;
+  const user = cashay.query(getAuthQueryString, getAuthedOptions(userId)).data.user;
   return {
     activity: state.userDashboardSettings.activity,
     nextPage: state.userDashboardSettings.nextPage,
-    userId: state.auth.obj.sub,
+    user,
+    userId: state.auth.obj.sub
   };
 };
 
 const validate = (values) => {
-  const schema = makeStep1Schema();
+  const schema = makeUpdatedUserSchema();
   return schema(values).errors;
 };
 
 @requireAuth
-@connect(mapStateToProps)
 @reduxForm({form: 'userSettings', validate})
+@connect(mapStateToProps)
```

Comment:
`@reduxForm` apparently was shallow-comparing the props it received. This stunk. It means that when our `user` sub-prop were changing, our view wasn't re-rendering. Poop on that, I just moved @connect down.